hack/quickstart/init-master.sh

```diff
@@ -23,7 +23,7 @@ function configure_etcd() {
         mkdir -p /etc/systemd/system/etcd-member.service.d
         cat << EOF > /etc/systemd/system/etcd-member.service.d/10-etcd-member.conf
 [Service]
-Environment="ETCD_IMAGE_TAG=v3.1.0"
+Environment="ETCD_IMAGE_TAG=v3.1.6"
```

Comment:
Yes. We should pin this in `glide.yaml`.
Bootkube seems to vendor latest etcd and etcd-operator version already. We don't need to update in this PR. I would like to pin in separate PR.